test/features/memory/create_test.rb

```diff
@@ -11,18 +11,17 @@
   scenario "the memory index has a link to a new memory form" do
     # when visiting the memory index
     visit memories_path
-
     # then a link to the create form should exist
-    page.find('#create-memory-button').click
+    #page.find('#create-memory-button').click
```

Comment:
just put in `click_on "New Memory"` for now and we can change these when we start building out the frontend